chart/templates/mayastor/obs/stats-deployment.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.obs.callhome.enabled }}
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Btw, can't we just add this as a separate container within the callhome deployment, is there any benefit to having this separate?

Reply:
addressed in PR #253